tests/Transports.AspNetCore.Tests/Transports.AspNetCore.Tests.csproj

```diff
@@ -8,8 +8,8 @@
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Castle.Core" Version="5.0.0" />
-    <PackageReference Include="GraphQL.SystemTextJson" Version="5.1.1" />
-    <PackageReference Include="GraphQL.NewtonsoftJson" Version="5.1.1" />
+    <PackageReference Include="GraphQL.SystemTextJson" Version="$(GraphQLVersion)" />
```

Comment:
I do not see any usages of GraphQLVersion in test.yml

Reply:
Like in auth/relay project I mean.

Reply:
No, this is just the version we are referencing project-wide, and is called `GraphQLVersion`, unlike `GraphQLTestVersion` that is used by auth/relay to provide testing for multiple versions against a the compiled package.

We could certainly update the CI scripts to provide additional testing in a similar manner as the auth/relay packages do.

Reply:
I can update the CI scripts in a subsequent PR.